python/applications/snake/snake_physics_node.py

```diff
@@ -0,0 +1,150 @@
+import time
+import sys, random, math
+import spacetime
+import argparse
+from uuid import uuid4
+from random import randint
+from multiprocessing import freeze_support
+from rtypes import pcc_set, dimension, primarykey
+from snake_datamodel import Snake, Apple, World, Direction, FRAMETIME
+from snake_bot import bot_execution
+from snake_visualizer_node import visualize
+from spacetime import Node
+import numpy as np
+
+WAIT_FOR_START = 5
+def accepting_players(dataframe, player_count):
+    #function for completing phase 1 of 'game_physics' function
+    #Deleting all the previous snakes:
+    # print("There are {} snakes.".format(len(dataframe.read_all(Snake))))
+    #dataframe.delete_all(Snake)
+
+    no_players = True
+    while no_players:
+        dataframe.checkout_await()
+        players = dataframe.read_all(Snake)
+        print (len(players))
+        if len(players) < player_count:
+            continue
+        return True
+
+def initializing_apple(dataframe):
+    #function for completing part of phase 2 of 'game_physics()' function
+    apple = Apple()
+    dataframe.add_one(Apple, apple)
+    dataframe.commit()
+
+def collision_with_apple(apple_position,score):
+    apple_position = [random.randrange(1, World.display_width),random.randrange(1, World.display_height)]
```

Comment:
reassignment of position, does not follow same logic as initialization of position.